app/main.py

```diff
@@ -1,13 +1,17 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
 
+def format_linter_error(error: dict) -> dict:
+    return [error:"code", "E501"]
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return [error:"code": "E501"]
```

Comment:
Similar to the previous function, there is a syntax error in the return statement. Ensure you use the correct dictionary syntax: `{'key': 'value'}`.